.cirrus.yml

```diff
@@ -448,6 +446,37 @@ osx_alt_build_task:
     always: *runner_stats
 
 
+# Build freebsd release natively on a FreeBSD VM.
+freebsd_alt_build_task:
+    name: "FreeBSD Cross"
+    alias: freebsd_alt_build
+    # Docs: ./contrib/cirrus/CIModes.md
+    only_if: |
```

Comment:
Minor thing (It's my fault):  Would be nice to have a comment here like `# Restricted to runs on 'main' due to runtime package installs.  TODO: Use pre-built CI VM images.`

Not worth re-pushing & re-running tests for this though.